RELEASE_NOTES.md

```diff
@@ -2,6 +2,66 @@
 
 While [CHANGELOG.md](./CHANGELOG.md) contains detailed documentation and links to all the source code changes in a given release, this document is intended to be aimed at a more comprehensible version of the contents of the release from the point of view of users of Refinery.
 
+## Version 2.9.0
+
+This release has two major features: one that improves memory consumption reporting, and one experimental feature for configuring trace locality mode.
+
+### Improved Memory Reporting
```

Comment:
```suggestion
### Improved Memory Usage
```

Recommend this changed wording, since the thing that users get here isn't a mechanism that reports memory better, but a deployment that uses memory more effectively.